src/components/ProductsHeader.tsx

```diff
@@ -1,56 +1,52 @@
-import { BsGridFill } from "react-icons/bs";
-import { FaListUl } from "react-icons/fa";
-import { AiOutlineDown } from 'react-icons/ai'
+import { BsGridFill } from "react-icons/bs"
+import { FaListUl } from "react-icons/fa"
+import { AiOutlineDown } from "react-icons/ai"
 
-import { DropdownContainer } from "./DropdownContainer";
-import { Checkbox, Input, RadioButton } from "./ui";
-import { Button } from "./ui/Button";
+import { DropdownContainer } from "./DropdownContainer"
+import { Checkbox, Input, RadioButton } from "./ui"
+import { Button } from "./ui/Button"
 
 interface navProps {
-  sortOption: { grid: boolean; list: boolean };
-  setSortOption: React.Dispatch<
-    React.SetStateAction<{ grid: boolean; list: boolean }>
-  >;
-  productsCount: number;
+  sortOption: { grid: boolean; list: boolean }
+  setSortOption: React.Dispatch<React.SetStateAction<{ grid: boolean; list: boolean }>>
+  productsCount: number
 }
-const ProductsHeader = ({
-  sortOption,
-  setSortOption,
-  productsCount,
-}: navProps) => {
+const ProductsHeader = ({ sortOption, setSortOption, productsCount }: navProps) => {
   return (
     <nav className="font-primary mb-8 grid w-full max-w-[1284px] grid-cols-1 items-center gap-x-8 px-8 pt-4 text-xs font-bold text-white tablet:grid-cols-[auto,auto,1fr,auto]">
       <div className="grid w-14 grid-cols-2 gap-x-2 ">
         <button
           onClick={() => setSortOption({ grid: true, list: false })}
-          className={`w-6 rounded-md ${sortOption.grid
-            ? "bg-black text-white"
-            : "scale-90 bg-white text-black"
-            }  stroke-black stroke-2  p-1 `}
+          className={`w-6 rounded-md ${
+            sortOption.grid ? "bg-black text-white" : "scale-90 bg-white text-black"
+          }  stroke-black stroke-2  p-1 `}
         >
           {<BsGridFill size={16} />}
         </button>
         <button
           onClick={() => setSortOption({ grid: false, list: true })}
-          className={`w-6 rounded-md ${sortOption.list
-            ? "bg-black text-white"
-            : "scale-90 bg-white  text-black"
-            }  stroke-black stroke-2  p-1 `}
+          className={`w-6 rounded-md ${
+            sortOption.list ? "bg-black text-white" : "scale-90 bg-white  text-black"
+          }  stroke-black stroke-2  p-1 `}
         >
```

Comment:
The Checkbox component was very specific to its use and there was a lot of styles going on with it so I didn't want to mess up other parts of the website. I use prettier but I might have to adjust settings since it is still using default settings.

Reply:
We use this .prettierrc settings
```
{
  "trailingComma": "none",
  "printWidth": 120,
  "semi": false,
  "tabWidth": 2
}
```

It should work for you - if its not let me know and try somehow fit it
In vs code settings you should have prettier and I think that this line (bracket same line) should help
![image](https://github.com/Nicitaa/22_aer/assets/39565703/d423120b-9860-4032-9d6f-c14c5ad488f8)

Have you installed prettier?
![image](https://github.com/Nicitaa/22_aer/assets/39565703/80ffa627-689e-4c43-a14b-cfbda03821c8)

Reply:
Resolved as 'prettier' is bull shit and some settings doesn't work on save
I create on my wish pre-commit hook which run 'pnpm prettier --write src/*' so we will have the same styling before commit